spec/models/import/transactions/actual_and_refund_csv_row_spec.rb

```diff
@@ -0,0 +1,583 @@
+        allow(csv_row).to receive(:field).with("Financial Year").and_return("fish")
+        row = described_class.new(csv_row)
+
+        expect(row.financial_year).to eql("fish")
+      end
+    end
+  end
+
+  describe "#receiving_organisation_name" do
+    it "returns the value from the csv row" do
+      csv_row = double(CSV::Row)
+      allow(csv_row).to receive(:field).with("Receiving Organisation Name").and_return("The Receiving Organisation")
+      row = described_class.new(csv_row)
+
+      expect(row.receiving_organisation_name).to eql("The Receiving Organisation")
+    end
+
+    context "when the receiving organisation name is empty" do
+      it "returns blank" do
+        csv_row = double(CSV::Row)
+        allow(csv_row).to receive(:field).with("Receiving Organisation Name").and_return(nil)
+        row = described_class.new(csv_row)
+
+        expect(row.receiving_organisation_name).to eql(:blank)
+      end
+    end
+  end
+
+  describe "#receiving_organisation_itai_reference" do
+    it "returns the value from the csv row" do
+      csv_row = double(CSV::Row)
+      allow(csv_row).to receive(:field).with("Receiving Organisation IATI Reference").and_return("IATI-123456")
+      row = described_class.new(csv_row)
+
+      expect(row.receiving_organisation_iati_reference).to eql("IATI-123456")
+    end
+
+    context "when the value is empty" do
+      it "returns blank" do
+        csv_row = double(CSV::Row)
+        allow(csv_row).to receive(:field).with("Receiving Organisation IATI Reference").and_return(nil)
+        row = described_class.new(csv_row)
+
+        expect(row.receiving_organisation_iati_reference).to eql(:blank)
+      end
+    end
+  end
+
+  describe "#receiving_organisation_type" do
+    context "when the type code is from the code list" do
+      it "returns the value from the row" do
+        csv_row = double(CSV::Row)
+        allow(csv_row).to receive(:field).with("Receiving Organisation Type").and_return("10")
+        row = described_class.new(csv_row)
+
+        expect(row.receiving_organisation_type).to eql("10")
+      end
+    end
+
+    context "when the type code is not from the code list" do
+      it "returns the value for validation later" do
+        csv_row = double(CSV::Row)
+        allow(csv_row).to receive(:field).with("Receiving Organisation Type").and_return("10000")
+        row = described_class.new(csv_row)
+
+        expect(row.receiving_organisation_type).to eql("10000")
+      end
+    end
+
+    context "when the receiving organisation type is empty" do
+      it "returns blank" do
+        csv_row = double(CSV::Row)
+        allow(csv_row).to receive(:field).with("Receiving Organisation Type").and_return(nil)
+        row = described_class.new(csv_row)
+
+        expect(row.receiving_organisation_type).to eql(:blank)
+      end
+    end
+  end
+
+  describe "#transaction_type" do
+    context "when the actual value is not zero and the refund value is zero" do
```

Comment:
What does it return when the actual value is not zero and the refund value is an empty string or nil?

I would expect it to return `:actual`.